osu.Game.Rulesets.Mania/Scoring/ManiaHealthProcessor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Rulesets.Mania.Scoring
+{
+    public class ManiaHealthProcessor : HealthProcessor
+    {
+        /// <summary>
+        /// The hit HP multiplier at OD = 0.
+        /// </summary>
+        private const double hp_multiplier_min = 0.75;
+
+        /// <summary>
+        /// The hit HP multiplier at OD = 0.
```

Comment:
fix the docs together with copying?

Reply:
What part of it am I fixing, the spelling? This is all going to be removed in a future PR anyway, so...

Reply:
The OD value were wrong in the original ScoreProcessor(should be 5 here). Sure to keep it if it'll be removed.

Reply:
Will keep it for now. See: https://github.com/ppy/osu/pull/7361 where this entire class has been removed.